techbuzz/src/main/java/com/sivalabs/techbuzz/security/WebSecurityConfig.java

```diff
@@ -31,7 +31,8 @@ public class WebSecurityConfig {
         "/registration",
         "/registrationStatus",
         "/verify-email",
-        "/c/**"
+        "/c/**",
+        "/resentVerification"
```

Comment:
**Minor nit pick:**
As the "resend' action not yet happened, I think we can name it as "resendVerification" instead of "resentVerification". Also, in other places please use "Resend" instead of "Resent".

Reply:
good catch. agree
Thanks updated